AttributeError: module 'typing' has no attribute 'NoReturn' with Python 3.5.3

I get this error when running the tests with Python 3.5.3.

```
tests/test_py3/test_aiohttpparser_async_functions.py:6: in <module>
    from webargs.aiohttpparser import parser, use_args, use_kwargs
webargs/aiohttpparser.py:72: in <module>
    class AIOHTTPParser(AsyncParser):
webargs/aiohttpparser.py:148: in AIOHTTPParser
    ) -> typing.NoReturn:
E   AttributeError: module 'typing' has no attribute 'NoReturn'
```

The docs say [`typing.NoReturn`](https://docs.python.org/3/library/typing.html#typing.NoReturn) was added in 3.6.5. However, [the tests pass on Travis](https://travis-ci.org/marshmallow-code/webargs/jobs/486701760) with Python 3.5.6.

--- a/webargs/aiohttpparser.py
+++ b/webargs/aiohttpparser.py
@@ -145,7 +145,7 @@ class AIOHTTPParser(AsyncParser):
         schema: Schema,
         error_status_code: typing.Union[int, None] = None,
         error_headers: typing.Union[typing.Mapping[str, str], None] = None,
-    ) -> typing.NoReturn:
+    ) -> "typing.NoReturn":
         """Handle ValidationErrors and return a JSON response of error messages
         to the client.
         """
@@ -163,7 +163,7 @@ class AIOHTTPParser(AsyncParser):
 
     def handle_invalid_json_error(
         self, error: json.JSONDecodeError, req: Request, *args, **kwargs
-    ) -> typing.NoReturn:
+    ) -> "typing.NoReturn":
         error_class = exception_map[400]
         messages = {"json": ["Invalid JSON body."]}
         raise error_class(

